reference/docs-conceptual/dev-cross-plat/create-feedback-provider.md

```diff
@@ -0,0 +1,682 @@
+---
+description: This article describes how to create a feedback provider.
+ms.date: 09/01/2023
+title: How to create a feedback provider
+---
+# How to create a feedback provider
```

Comment:
I think this article should also point to the https://github.com/PowerShell/command-not-found repo as a sample for developers. Also point to the module in gallery: https://www.powershellgallery.com/packages/command-not-found/0.1.0.